src/v2i-hub/MessageLoggerPlugin/src/MessageLoggerPlugin.cpp

```diff
@@ -652,30 +582,20 @@ std::string MessageLoggerPlugin::GetCurDateTimeStr()
 // This method does not need to be overridden if the plugin does not want to use the main thread.
 int MessageLoggerPlugin::Main()
 {
-	PLOG(logDEBUG) << "Starting MessageLoggerplugin...";
+	PLOG(logINFO) << "Starting MessageLoggerplugin...";
 
-	uint msCount = 0;
 	while (_plugin->state != IvpPluginState_error)
 	{
-		PLOG(logDEBUG4) << "MessageLoggerPlugin Sleeping" << endl;
-
-		this_thread::sleep_for(chrono::milliseconds(1000));
-
-
+		PLOG(logDEBUG4) <<"MessageLoggerPlugin: Sleeping 5 minutes";
+		this_thread::sleep_for(chrono::milliseconds(300000));
```

Comment:
I know this is unrelated to your fix but may want to rethink this in a future issue.

Reply:
Yeah, this introduces unwanted an unwanted latency. I think we removed the same thing in the SPAT plugin. I'm planning on having a branch to remove this from other plugins.